Gemfile.lock

```diff
@@ -10,22 +10,22 @@ GEM
   remote: https://rubygems.org/
   specs:
     eventmachine (1.2.7)
-    faye-websocket (0.10.7)
+    faye-websocket (0.10.9)
       eventmachine (>= 0.12.0)
       websocket-driver (>= 0.5.1)
     mustermann (1.0.3)
-    rack (2.0.7)
-    rack-protection (2.0.5)
+    rack (2.0.8)
+    rack-protection (2.0.7)
       rack
-    sinatra (2.0.5)
+    sinatra (2.0.7)
       mustermann (~> 1.0)
       rack (~> 2.0)
-      rack-protection (= 2.0.5)
+      rack-protection (= 2.0.7)
       tilt (~> 2.0)
-    tilt (2.0.9)
-    websocket-driver (0.7.0)
+    tilt (2.0.10)
+    websocket-driver (0.7.1)
```

Comment:
### [websocket-driver](https://github.com/faye/websocket-driver-ruby)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.7.0 → 0.7.1

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **14** commits. See the full changes on [the compare page](https://github.com/faye/websocket-driver-ruby/compare/0.7.0...0.7.1).

These are the first 10 commits:
- (2ec898c) [If any driver encounters a validation error in the request h…](https://github.com/faye/websocket-driver-ruby/commit/2ec898c6a10f0b575f01a5e418652d05e27c3794)
- (1147631) [If any header used by Hybi is present, then pick Hybi, and l…](https://github.com/faye/websocket-driver-ruby/commit/11476318eacad0bd125456af05d143cfa9bae783)
- (b14488a) [Bump the Ruby versions for Travis.](https://github.com/faye/websocket-driver-ruby/commit/b14488aac51b7e3be726c74ecc7fcd8cb14819e3)
- (ec94a21) [I think you have to use jruby-head instead of jruby-9 now.](https://github.com/faye/websocket-driver-ruby/commit/ec94a219e91a0cf7968c076e67060c48d458b257)
- (ee39af8) [Test on Ruby 2.5.0.](https://github.com/faye/websocket-driver-ruby/commit/ee39af83d03ae3059c775583e4c4b291641257b8)
- (2c19210) [Switch license to Apache 2.0.](https://github.com/faye/websocket-driver-ruby/commit/2c19210a694c7c8988bfc2198bef7342f08c9ae6)
- (5b957c6) [Update Travis target versions.](https://github.com/faye/websocket-driver-ruby/commit/5b957c66607599ba100724f478366ef07f549602)
- (8a6f9ef) [Fix an uninitialised variable warning.](https://github.com/faye/websocket-driver-ruby/commit/8a6f9ef737c02ef7c773e1aa2f2074aba50a26f9)
- (77ae543) [Reformat the C and Java native extension modules.](https://github.com/faye/websocket-driver-ruby/commit/77ae5437842b8a23e466f90e09cdb91b745267dc)
- (6b519ee) [Depend on Rake < 12.3 if we're running on Ruby < 2.0.](https://github.com/faye/websocket-driver-ruby/commit/6b519eecc30423d5aa9b5b84f800e63d0d639ff2)


</details>